stdlib/seq.ml

```diff
@@ -28,22 +28,22 @@ let return x () = Cons (x, empty)
 let cons x next () = Cons (x, next)
 
 let rec append seq1 seq2 () =
-  match seq1() with
-  | Nil -> seq2()
+  match seq1 () with
+  | Nil -> seq2 ()
```

Comment:
These whitespace diffs are unrelated; there's nothing wrong with them, but there's also no reason to include them in the present PR; can you revert them? Thanks!

Reply:
Done.